en/components/navdrawer.md

```diff
@@ -316,6 +316,65 @@ After all the steps above are completed, your app should look like that:
 </code-view>
 
 
+<div class="divider--half"></div>
+
+## Hierarchical Navigation
+
+To create a multi-level hierarchical navigation using the `IgxNavigationDrawerComponent`, you can use the `[IgxTreeComponent](tree.md)` in the `igxDrawer` template. The tree can be constructed directly from your application `Routes` object. Here's how this can be acieved:
+
+```html
+<igx-nav-drawer [isOpen]="true" [enableGestures]="true" width="280px">
+    <ng-template igxDrawer>
+        <igx-tree>
+            <igx-tree-node *ngFor="let route of routes">
+                <a igxTreeNodeLink [routerLink]="route.path">{{ route.data?.displayName }}</a>
+                <igx-tree-node *ngFor="let child of route.children">
+                    <a igxTreeNodeLink [routerLink]="[route.path, child.path]">{{ child.data?.displayName }}</a>
+                </igx-tree-node>
+            </igx-tree-node>
+        </igx-tree>
+    </ng-template>
+</igx-nav-drawer>
+```
+
+> [!NOTE]
+> In this example, we're not using the `igxDrawerItem`, instead we're populating directly with custom `igxDrawer` content, in this case using an `igx-tree`.
```

Comment:
Minor nit: You can probably work this into the intro of the section, something about "any content can be used in the template", though that's likely a personal preference of more text vs info box.

Reply:
PS: Actually, the "While any content can be provided" is already in the Usage section, so perhaps no point explaining it again I guess